smartapps/smartthings/logitech-harmony-connect.src/logitech-harmony-connect.groovy

```diff
@@ -813,7 +813,7 @@ def deviceHandler(evt) {
 		}
 	} else if (deviceInfo) {
 		if (deviceInfo.callbackUrl) {
-			sendToHarmony(evt, deviceInfo.callbackUrl)
+			sendToHarmonyCloud(evt, deviceInfo.callbackUrl)
```

Comment:
I know you could at least parse out the host with:

```
new URI("http://10.0.1.1/foo").host // foo
```

As for knowing if an IP address is in the host rather than some DNS name, you could try using a Groovy regex for that.